charts/puris-backend/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: puris-backend
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
```

Comment:
Please add missing license header, `sources` and `home` key, [TRG 5.02](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-2#the-chartyaml-file).

Reply:
Thanks a lot for the review and the suggestions! I used other repositories currently available in Eclipse Tractus-X as a reference, like the _Item Relationship Service_ or the _Portal_. Since they all haven't included `sources`, `home` etc., it isn't included here either.
Nevertheless, I updated the `Chart.yaml` to include everything mentioned in the current guidelines to be compatible with it.